modal/functions.py

```diff
@@ -1394,6 +1391,32 @@ async def coro():
                 obj.enter()
                 return fun(*args, **kwargs)
 
+    @synchronizer.no_input_translation
+    @live_method
+    async def _experimental_spawn(self, *args: P.args, **kwargs: P.kwargs) -> "_FunctionCall[ReturnType]":
+        """[Experimental] Calls the function with the given arguments, without waiting for the results.
+
+        This experimental version of the spawn method allows up to 1 million inputs to be spawned.
+
+        Returns a `modal.functions.FunctionCall` object, that can later be polled or
+        waited for using `.get(timeout=...)`.
+        Conceptually similar to `multiprocessing.pool.apply_async`, or a Future/Promise in other contexts.
+
+        *Note:* `.spawn()` on a modal generator function does call and execute the generator, but does not currently
+        return a function handle for polling the result.
+        """
+        self._check_no_web_url("spawn")
```

Comment:
Nit: string input to _check_no_web_url should probably be `"_experimental_spawn"` here